backend/app/running_config/running_config_service.py

```diff
@@ -25,4 +26,4 @@ def get_configs_for_upload(self, decrypted_xml: dict) -> TopologyModel:
         except Exception as exc:
             logging.error(f"Error occurred while extracting configs from xml: {str(exc)}")
 
-        return TopologyModel(topology=topology_config)
+        return TopologyModel(topology=topology_config, name='')
```

Comment:
Jeśli chcemy nadawać nazwy tym topologią to możesz od razu dodać to jako np. body. Pytanie tylko czy nazywanie tych topologii jest nam potrzebne?

A no i wygląda jakby w obecnej postaci miało się to wywalić, bo nie ma takiego pola w TopologyModel, ale nie zdziwię się jeśli jednak python to łyka... bo to python xd

Reply:
Już jest name, Bartek dodał, bo w mappingu było mu potrzebne i ma ogarnąć 
https://github.com/AntuanW/AGH-Engineering-Thesis/blob/dev/backend/app/models/topology.py

Reply:
My bad, dzięki za info bo nie zwróciłem uwagi nawet.